install/migrations/update_10.0.x_to_11.0.0/assets.php

```diff
@@ -114,7 +114,7 @@
     }
 }
 
-$ADDTODISPLAYPREF['Glpi\\Asset\\AssetDefinition'] = [2, 3, 4, 5, 6];
+$ADDTODISPLAYPREF[\Glpi\Asset\AssetDefinition::class] = [2, 3, 4, 5, 6];
```

Comment:
Won't that be an issue if class is renamed or deleted in the future?
I'm totally OK with this kind of change in the code itself; but maybe not in update scripts.

Reply:
I added a skip instruction and reverted these changes.